.gitignore

```diff
@@ -0,0 +1 @@
+chaincode/ping/ping
```

Comment:
Why is this required? (Thinking of replacing gitignore with https://github.com/github/gitignore/blob/master/Go.gitignore but let me know if this entry should stay.)